api/v1beta1/policyautomation_types.go

```diff
@@ -52,6 +56,22 @@ type AutomationDef struct {
 	TowerSecret string `json:"secret"`
 	// JobTTL sets the time to live for the Kubernetes AnsibleJob object after the Ansible job run has finished.
 	JobTTL *int `json:"jobTtl,omitempty"`
+	// The maximum number of violating cluster contexts that will be provided to the Ansible job as extra variables.
+	// When policyViolationContextLimit is set to 0, it means no limit.
+	// The default value is 1000.
+	PolicyViolationContextLimit *uint `json:"policyViolationContextLimit,omitempty"`
+}
+
+// ViolationContext defines the non-compliant replicated policy information
+// that is sent to the AnsibleJob through extra_vars.
+type ViolationContext struct {
+	TargetClusters  []string `json:"targetClusters,omitempty" ansibleJob:"target_clusters"`
```

Comment:
I know this is never empty, but it'd be good to be consistent.
```suggestion
	TargetClusters  []string `json:"targetClusters" ansibleJob:"target_clusters"`
```